example/lib/pages/basic_page.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+class BasicPage extends StatelessWidget {
+   final Widget widget;
+
+  const BasicPage({
+    Key? key,
+    required  this.widget,
```

Comment:
Instead to call this "widget", you can change with a more descriptive variable name like bodyWidget, so when you are going to create a BasicPage you can see something like this: BodyPage( bodyWidget: ... ), it's tell you where you are going to use the Widget that you pass as parameter.

Reply:
done